packages/cli/src/credentials/credentials.service.ts

```diff
@@ -108,23 +123,38 @@ export class CredentialsService {
 				credentials = credentials.map((c) => this.ownershipService.addOwnedByAndSharedWith(c));
 			}
 
-			if (options.includeScopes) {
+			if (includeScopes) {
 				credentials = credentials.map((c) =>
 					this.roleService.addScopes(c, user, projectRelations!),
 				);
 			}
 
+			if (includeData) {
+				credentials = credentials.map((c: CredentialsEntity & ScopesField) => {
+					if (c.scopes.includes('credential:update')) {
+						return {
+							...c,
+							data: this.decrypt(c),
```

Comment:
every `this.decrypt` return value should also go through `this.redact` to make sure that we don't ever send any secrets back to the frontend.
Considering that this mistake has happened in previous PRs, I think we should also switch to redacting by default in `this.decrypt`. 
I don't think any of the credentials related code on these controllers and services needs access to the raw secrets. 
redacting by default would also simplify this code a tiny bit.